scripts/check_compliance.py

```diff
@@ -426,73 +426,123 @@ class Codeowners(ComplianceTest):
     _name = "Codeowners"
     _doc  = "https://help.github.com/articles/about-code-owners/"
 
-    def parse_codeowners(self, git_root, codeowners):
-        all_files = []
+
+    def ls_owned_files(self, codeowners):
+        """Returns a OrderedDict mapping git patterns from the CODEOWNERS file
```

Comment:
*an

Reply:
?

Reply:
"an OrderedDict" instead of "a OrderedDict", since 'O' is a vowel. :)